rackspace.yaml

```diff
@@ -1,6 +1,6 @@
 schema-version: 1.0.0
 application-family: Databases
-application-name: BigData
+application-name: Hadoop
 flavor: Hadoop
```

Comment:
What would you change this to? What would this be for the other templates as well?